### DISCUSSION TITLE
Свойства сотрудника необходимые лаб группе

### DISCUSSION THREAD

**Discussion by ai-fhir** [fhir-ru/core#77] [Административная РГ]
Я не уверен, что Practitioner/PractitionerRole относятся к Административной рабочей группе, а не к клинической, но мы можем поменять категорию, если она задана неверно.

Диаграмма описывает поля сотрудника (Practitioner/PractitionerRole), которые необходимы лабораторной группе, и должны быть учтены в базовом профиле. 

![practitioner_lab](https://user-images.githubusercontent.com/93438901/152655368-9f98bf70-33d9-4029-9850-9cb2109f1cc7.jpg)

cc @avzviagin

**Comment by eugenekogan-spb** [comment]
Сотрудник - информация о том, что данная персона работает в данной должности в данной медицинской организации. Возможно, в течение периода.
В связи с этим вопросы:
- какой адрес сотрудника имеется в виду?
- верно ли, что контактные данные - это рабочий контакт сотрудника именно в этой организации?
- не следует ли добавить дату начала и завершения, чтобы было проще регистрировать факт увольнения?

**Comment by eugenekogan-spb** [comment]
Еще вопрос: коды должностей частные медицинские организации ведут по по справочнику должностей НСИ ЕГИСЗ или используют самостоятельно поддерживаемые справочники?

**Comment by ai-fhir** [comment]
Дополнительно прикладываю примеры CDA, чтобы можно было посмотреть используемые там значения

[Протокол Лаб Исследования CDA](https://github.com/fhir-ru/pli/blob/main/assets/CDA_%D0%9F%D0%A0%D0%9E%D0%A2%D0%9E%D0%9A%D0%9E%D0%9B_%D0%9B%D0%90%D0%91%D0%9E%D0%A0%D0%90%D0%A2%D0%9E%D0%A0%D0%9D%D0%9E%D0%93%D0%9E_%D0%98%D0%A1%D0%A1%D0%9B%D0%95%D0%94%D0%9E%D0%92%D0%90%D0%9D%D0%98%D0%AF_%D0%A04.docx)

[Протокол лабораторного исследования пример максимальный xml](https://github.com/fhir-ru/pli/blob/main/assets/%D0%9F%D1%80%D0%BE%D1%82%D0%BE%D0%BA%D0%BE%D0%BB_%D0%BB%D0%B0%D0%B1%D0%BE%D1%80%D0%B0%D1%82%D0%BE%D1%80%D0%BD%D0%BE%D0%B3%D0%BE_%D0%B8%D1%81%D1%81%D0%BB%D0%B5%D0%B4%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F_%D0%BF%D1%80%D0%B8%D0%BC%D0%B5%D1%80_%D0%BC%D0%B0%D0%BA%D1%81%D0%B8%D0%BC%D0%B0%D0%BB%D1%8C%D0%BD%D1%8B%D0%B9.xml)
